07-sampling.Rmd

```diff
@@ -1064,33 +1062,245 @@ Comment on the representativeness of the following *sampling methodologies*:
 ```
 
 
-## Central Limit Theorem {#sampling-conclusion-central-limit-theorem}
+## A theoretical approach and the Central Limit Theorem
 
-This chapter began with (virtual) access to a large bowl of balls (our population) and a desire to figure out the proportion of red balls. Despite having access to this population, in reality, you almost **never** will have **access to the population,** either because the population is too large, ever changing, or too expensive to take a census of. Accepting this reality means accepting that we need to use _statistical inference._ 
+We now introduce a simple theoretical approach to understanding some important properties of the sample proportion. To do this we modify the bowl activity slightly. Instead of using a shovel to select all 25 balls at once, we randomly select one ball at a time. If the ball is red we call it a success and record a 1 (one); if it is not red we call it a failure and record a 0 (zero). Then, we return the ball to the bowl so the proportion of red balls in the bowl doesn't change. 
+This process is called a trial or a Bernoulli trial in honor of Jacob Bernoulli, a 17th-century mathematician who used it already back in those times. 
+Getting a sample of 25 balls is running 25 trials and obtaining 25 numbers, ones or zeros, representing whether we have observed red balls or not on each trial, respectively. 
+The average of these 25 numbers is the sample proportion. 
 
-In Section \@ref(terminology-and-notation), we stated that "statistical inference is the act of making a guess about a population using a sample." But how do we _do_ this inference? In the previous section, we defined the _sampling framework_ only to state that in reality we take one large sample, instead of many samples as done in the sampling framework (which we modeled physically by taking many samples from the bowl). 
+### Random Variables
 
-In reality, we take *only one* sample and use that *one* sample to make statements about the population parameter. This ability of making statements about the population is allowable by a famous theorem, or mathematically proven truth, called the \index{Central Limit Theorem} *Central Limit Theorem*. What you visualized in Figures \@ref(fig:comparing-sampling-distributions) and \@ref(fig:comparing-sampling-distributions-2) and summarized in Tables \@ref(tab:comparing-n) and \@ref(tab:comparing-n-2) was a demonstration of this theorem. It loosely states that when sample means are based on larger and larger sample sizes, the sampling distribution of these sample means becomes both more and more normally shaped and more and more narrow.
+It is useful to represent a trial as a random variable that takes the value of 1 if the ball is red or the value of 0 if the ball is not red. 
```

Comment:
@avaldivi6 Maybe explain a little more both of the words "random" and "variable" here from a novice perspective? We don't know whether a given ball chosen will be red or not red so the event in question is random. It will likely vary from one selection to another. Something along those lines.

Reply:
@avaldivi6 It looks like this wasn't updated. Would you like to discuss further on our next call?

Reply:
@ismayc Yes, let's talk about this